egs2/ksponspeech/asr1/conf/train_asr_conformer8_n_fft512_hop_length256.yaml

```diff
@@ -0,0 +1,72 @@
+# Trained with Tesla V100-SXM2(32GB) x 8 GPUs. It takes about 1.5 days.

```

Comment:
I recommend you to move this file under `local/tuning`
Then, use a symlink, e.g., `conf/train_asr_conformer.yaml --> conf/tuning/train_asr_conformer8_n_fft512_hop_length256.yaml` like the other recipes.

Reply:
Fixed as suggested.